nextflow_schema.json

```diff
@@ -199,15 +199,15 @@
                     "help_text": "`Sarek` will build missing indices automatically if not provided. Set `--bwa false` if indices should be (re-)built.\nIf `DragMap` is selected as aligner, it is recommended to skip baserecalibration with `--skip_tools baserecalibrator`. See [here](https://gatk.broadinstitute.org/hc/en-us/articles/4407897446939--How-to-Run-germline-single-sample-short-variant-discovery-in-DRAGEN-mode) for more info.\n",
                     "hidden": true
                 },
-                "save_bam_mapped": {
+                "save_mapped": {
                     "type": "boolean",
                     "fa_icon": "fas fa-download",
-                    "description": "Save mapped BAMs.",
-                    "help_text": "If the parameter `--split-fastq` is used, the sharded bam files are merged before saving them."
+                    "description": "Save mapped files.",
+                    "help_text": "If the parameter `--split-fastq` is used, the sharded bam files are merged and converted to CRAM before saving them."
                 },
                 "save_output_as_bam": {
                     "type": "boolean",
-                    "description": "Saves output from Markduplicates & Baserecalibration as BAM file instead of CRAM",
+                    "description": "Saves output from BWA (if `--save_mapped`), Markduplicates & Baserecalibration as BAM file instead of CRAM",
```

Comment:
```suggestion
                    "description": "Saves output from mapping (if `--save_mapped`), Markduplicates & Baserecalibration as BAM file instead of CRAM",
```